File: Projects/JavaProject/src/com/Multithreading/ThreadExample.java
package com.Multithreading;

//Monitoring system health in the background, such as CPU usage or memory allocation.

class MyThread extends Thread {
    @Override
    public void run() {
        System.out.println("Thread is running: " + Thread.currentThread().getName());
    }
}

public class ThreadExample {
    public static void executeThreadExample() {
        MyThread thread = new MyThread();
        thread.start(); // Starts the thread
    }

    public static void main(String[] args) {
        executeThreadExample();
    }
}

File: Projects/PythonProject/Multithreading/ThreadExample.py
import threading

class MyThread(threading.Thread):
    def run(self):
        print(f"Thread is running: {threading.current_thread().name}")

def execute_thread_example():
    thread = MyThread()
    thread.start()  # Starts the thread

if __name__ == "__main__":
    execute_thread_example()
